app/views/comment_mailer/send_notification_email.html.erb

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+    </head>
+    <body>
+    <div style="width: 600px; height: 800px; margin: 0 auto;">
+      <div>
+        <table style="padding-bottom: 20px; width: 100%; ">
+        <tr>
+          <td>
+          <%= image_tag attachments['banner-header.png'].url, style: "width: 72px; height: 92px" %>
+          </td>
+          <td style="font-family: '.HelveticaNeueDeskInterface-Regular'; font-size: 12px; color: #9C9997; line-height: 10px; text-align: right; vertical-align: top; padding-top: 20px">You're signed up for instant activity notifications. Change your settings.
+          </td>
+        </tr>
+        </table>
+        <h1 style="font-family: HelveticaNeue-Bold; font-size: 30px; color: #331A48; line-height: 26px; margin-bottom: 0">Someone has replied to your post!</h1>
+        <p style="font-family: HelveticaNeue; font-size: 18px; color: #565656; line-height: 15px; margin-top: 5px;"><a href=<%= @url %> style="font-family: HelveticaNeue-Bold; font-size: 19px; color: #F72C54; line-height: 26px"><%= @user.username %></a> has replied to your post <a href=<%= @url %> style="font-family: HelveticaNeue-Bold; font-size: 18px; color: #F72C54; line-height: 15px;" ><%= @post.title %></a></p>
+        <p style="font-family: HelveticaNeue; font-size: 14px; color: rgba(87,83,80,0.60); line-height: 21px;">Hey there! I have some spare time over the next few weekends. You can see some of my <br/> work at <a href="https://www.rubygamedev.com/" style="color: black;"> www.rubygamedev.com</a>. I'm also connected to a really rad character designer if you're<br/> in need of one of those too.</p><br/><br/>
```

Comment:
This not appear in the design that you sent us  in Sketch, can you updated the mockup?